app/api/services/evidence.py

```diff
@@ -83,6 +84,112 @@ def get_previous_submitted_evidence_for_supplier_and_domain(self, evidence_id, d
         evidence = query.first()
         return evidence
 
+    def get_data(self, evidence_id):
```

Comment:
Is this function name too generic?  Would something like `get_submitted_evidence` be a better description of what this function is doing?

Reply:
I have changed it to `get_approved_evidence` since there is already a function called `get_submitted_evidence` in  the evidence service class.